### ISSUE TITLE
UI-theme

### ISSUE THREAD

**Issue by pulkit-30** [ElixirTechCommunity/Nexus_Website#5]
ref: https://colorhunt.co/

add theme colors as root-variables in css

```
:root {
  --primary-color: #1e90ff;
  --secondary-color: #ffffff;
  .
  .
  .

}

```

**Comment by Anv3sh** [comment]
ok 👍🏻

**Comment by Anv3sh** [comment]
A reference 
![image](https://user-images.githubusercontent.com/51405870/184435840-053565cd-e84d-4a41-90f5-3666f56cd510.png)

**Comment by Anv3sh** [comment]
#17

**Comment by pulkit-30** [comment]
Fixed #19
